### PR TITLE
Test and support python 3.9 and 3.10

### PR DESCRIPTION
**Pull request by nikolaik**
We are planning to use your lib python 3.9 and wanted to make sure it is tested on newer pythons

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.6, 3.7, 3.8]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
Comment (mammothb) [inline .github/workflows/tests.yml:13]:
```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]

```
I still want to support Python 3.6

File: setup.cfg [hunk 2]
```diff
@@ -19,9 +19,10 @@ classifiers =
     License :: OSI Approved :: MIT License
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
```
Comment (mammothb) [inline setup.cfg:22]: Similar to above, I still want to support 3.6

**Review by mammothb — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (nikolaik) [reply .github/workflows/tests.yml]: of course, added it back now d2fe39b1aa530378d1b0f27555d4d6b5011605ab My reasoning was that it hit [end of life ](https://devguide.python.org/devcycle/#end-of-life-branches) just before new year

**Review by mammothb — APPROVED** [review]

**Comment by nikolaik** [comment]
Thank you ❤️

**Pull request merged**